pkg/config/kubeClient.go

```diff
@@ -53,9 +54,18 @@ func getKubeConfig() *rest.Config {
 
 // Get kubernetes client for discovering resource types.
 func GetDiscoveryClient() *discovery.DiscoveryClient {
-	discoveryClient, err := discovery.NewDiscoveryClientForConfig(getKubeConfig())
+	discoveryClient, err := discovery.NewDiscoveryClientForConfig(GetKubeConfig())
 	if err != nil {
 		glog.Fatal("Cannot Construct Discovery Client From Config: ", err)
 	}
 	return discoveryClient
 }
+
+func GetKubeClient() *kubernetes.Clientset {
```

Comment:
This was previously used code? Commented out and saved for later? It's finally later! Good move, we need this method now.